docs/reference/search/request/scroll.asciidoc

```diff
@@ -175,3 +175,88 @@ curl -XDELETE localhost:9200/_search/scroll \
      -d 'c2Nhbjs2OzM0NDg1ODpzRlBLc0FXNlNyNm5JWUc1,aGVuRmV0Y2g7NTsxOnkxaDZ'
 ---------------------------------------
 
+
+==== Sliced Scroll
+
+For scroll queries that return a lot of documents it is possible to split the scroll in multiple slices which
+can be consumed independently:
+
+[source,js]
+--------------------------------------------------
+curl -XGET 'localhost:9200/twitter/tweet/_search?scroll=1m' -d '
+{
+    "slice": {
+        "id": 0, <1>
+        "max": 2 <2>
+    },
+    "query": {
+        "match" : {
+            "title" : "elasticsearch"
+        }
+    }
+}
+'
+
+curl -XGET 'localhost:9200/twitter/tweet/_search?scroll=1m' -d '
+{
+    "slice": {
+        "id": 1,
+        "max": 2
+    },
+    "query": {
+        "match" : {
+            "title" : "elasticsearch"
+        }
+    }
+}
+'
+--------------------------------------------------
+
+<1> The id of the slice
+<2> The maximum number of slices
+
+The result from the first request returned documents that belong to the first slice (id: 0) and the result from the
+second request returned documents that belong to the second slice. Since the maximum number of slices is set to 2
+ the union of the results of the two requests is equivalent to the results of a scroll query without slicing.
+By default the splitting is done on the _uid field and the following formula is used:
+
+`slice(doc) = floorMod(hashCode(doc._uid), max)`
+
+Each scroll is independent and can be processed in parallel like any scroll request.
+
+NOTE: The slice filter is very slow on the first calls, it has a cost of O(N) per request (in each sliced scroll) where
+ N is the number of documents. After few calls the filter should be cached and subsequent calls should be faster.
```

Comment:
we should totally document that the memory cost of the default here is 1 bit per document * num slices * shards? everytime I look at it it's more scary I don't know if we should keep it